phone_actions.rb

```diff
@@ -0,0 +1,15 @@
+class PhoneActions
+
+  def initialize(username, password)
+    @actions_url = "https://#{username}:#{password}@api.<made_up>.com/actions/"
+  end
+
+  def call!(number, message)
+    JSON.parse(RestClient.post("@actions_url/#{text}", {message: message, number: number}))
```

Comment:
Did you mean to include the variable in the string literal so that `RestClient` will receive the full URL?

```suggestion
    JSON.parse(RestClient.post("${@actions_url}/#{text}", {message: message, number: number}))
```

Reply:
For both this line and L12